CIPs/cip-110.md

```diff
@@ -3,7 +3,7 @@ cip: 110
 title: Ceramic Anchor Contract
 author: Sergey Ukustov <sergey@ukstv.me>
 discussions-to: https://github.com/ceramicnetwork/CIP/issues/110
-status: Review
+status: Final
```

Comment:
I think some of what's in this CIP doesn't match exactly what was built.  I assume once we mark something as Final we aren't supposed to change it, so should we update this CIP to be in alignment with what was built before we finalize it?

Some things I noticed:
1. The anchor event is called `DidAnchor` in this doc, but is actually called `anchorDagCbor` in reality.
2. The "Backwards Compatibility" section talks about adding `version: 1` to anchor commits, which we didn't do.  Instead we added `txType: "f(bytes32)"` to anchorProofs.

There may be more inconsistencies, I didn't check everything, that's just what I noticed from a quick scan

Reply:
Yeah, we should update them to the extent that we can. Will take a pass here when I have time.